exercises/csharp/TriviaGame/TriviaGame/GameModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TriviaGame
+{
+    public class GameModel
```

Comment:
I can't see where this GameModel class is actually used elsewhere (might be missing it though)